scripts/mk/variables.mk

```diff
@@ -16,6 +16,9 @@ COMPOSE_FILE ?= $(PROJECT_DIR)/deployments/docker-compose.yaml
 
 CONTAINER_IMAGE_BASE ?= quay.io/$(firstword $(subst +, ,$(QUAY_USER)))/$(APP_NAME)-$(APP_COMPONENT)
 
+GO_VERSION ?= 1.20
```

Comment:
I like this to reduce the burden when upgrading the golang version; thanks! :+1:

Reply:
I suggest to add a note at `go.mod` in the base repository to address where to update when upgrading the golang version for the future.

Reply:
`go.mod` is auto-generated and will be overwritten